harmoni_detectors/harmoni_stt/README.md

```diff
@@ -1,7 +1,25 @@
 # HARMONI STT
 
 ## Usage
+Using the DeepSpeech service:
+
+Note: Run get_deepspeech_models.sh before launching or testing this service.
```

Comment:
I would add this: ""This script should be run from the HARMONI directory in order to place the models in a parallel directory"" into the README